controller/src/main/java/org/jboss/as/controller/capability/registry/RequirementRegistration.java

```diff
@@ -29,6 +37,7 @@
  */
 public class RequirementRegistration {
 
+    protected final Map<PathAddress, RegistrationPoint> registrationPoints = new LinkedHashMap<>();
```

Comment:
Can this be private? I didn't see an obvious external use, although checking that in github is not an exact thing. ;)